minvws/nl-kat-coordination
# Feature/add boefje scheduling fields

### Changes

This adds the two fields described in the issue: `cron` and `interval`.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3527

### Demo

-

### QA notes

This change should not break the boefjes/normalizers. After creating a variant of a boefje, you could take a look at the fields and change them through the KATalogus [docs](http://localhost:8003/docs#/plugins/update_boefje_v1_organisations__organisation_id__boefjes__boefje_id__patch) to verify this change works at the API level.

---

### Code Checklist




---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
